e2e_tests/tests/cluster/test_workspace_org.py

```diff
@@ -588,3 +589,66 @@ def test_workspaceid_set() -> None:
 
         cmd = bindings.get_GetCommand(admin_session, commandId=cmd.id).command
         assert cmd.workspaceId == workspace.id
+
+
+@pytest.mark.e2e_cpu_rbac
+@pytest.mark.e2e_cpu
+def test_workspace_members() -> None:
+    """set up workspace with users, user-groups, and roles, and test list-member cli command"""
+    test_user: List[bindings.v1Users] = []
+    test_groups: List[bindings.v1Groups] = []
+    test_exp: List[str] = ["User/Group Name | User/Group | Role Name"]
+
+    try:
+        admin_sess = api_utils.admin_session()
+
+        # Add a test workspace.
+        workspace_name = api_utils.get_random_string()
+        create_workspace_cmd = ["det", "workspace", "create", workspace_name]
+        detproc.check_call(admin_sess, create_workspace_cmd)
+
+        # Create test users (3) and groups (2) and assign to workspace.
+        roles = ["Editor", "Viewer", "WorkspaceAdmin", "EditorRestricted", "ModelRegistryViewer"]
```

Comment:
Do we have some `enum` defined or a better way to access them? I couldn't find.

Reply:
not that I know of. but another suggestion around here would be to use this utility instead which directly uses our apis to set up (i think) what you're tying to set up https://github.com/determined-ai/determined/blob/769d600150439baca9f920ba4057f056d7d7a3d4/e2e_tests/tests/cluster/test_rbac.py#L25

Reply:
I would prefer keeping the code same to maintain the consistency while creating the group, their roles and assigning to the workspace.
Thanks for suggesting though.